src/Sarif.UnitTests/TestData/SarifVersionOneToCurrentVisitor/ExpectedOutputs/OneRunWithRules.sarif

```diff
@@ -70,7 +70,8 @@
               "text": "Rent internal rebellion competence biography photograph."
             },
             "configuration": {
-              "defaultLevel": "note",
+              "enabled" :  false,
```

Comment:
>"enabled" :  false, [](start = 14, length = 19)

bug in transformer logic didn't always flow enabled == false state properly, now fixed